nat-lab/tests/mesh_api.py

```diff
@@ -273,9 +275,14 @@ def config_dynamic_nodes(self, node_configs: List[bool]) -> Tuple[Node, ...]:
                 public_key=str(public),
                 is_local=is_local,
             )
-            # TODO correct subnet when we'll decide about the range
-            self.assign_ip(node.id, f"100.64.33.{node_idx}")
-            self.assign_ip(node.id, f"fd00::dead:{node_idx}")
+            ipv4 = (
+                f"100.{random.randint(65, 127)}.{random.randint(0, 255)}.{random.randint(8, 255)}"
+            )
+            ipv6 = (
+                f"fc74:656c:696f:0:{format(random.randint(0, 0xFFFF), 'x')}:{format(random.randint(0, 0xFFFF), 'x')}:{format(random.randint(0, 0xFFFF), 'x')}:{format(random.randint(0, 0xFFFF), 'x')}"
```

Comment:
`:0:` can be changed to `::` , last segment should have range of  `(8, 0xFFFF)` to account for reserved addresses.

Reply:
updated, but left `:0:` since there is no proper ipv6 string comparison... will leave it for future implementation